### PR TITLE
Create TheHome and TheHeader 

### LINKED ISSUE #28: Create Home Component

### LINKED ISSUE #29: Create Header Component

### PR DESCRIPTION
**Pull request by nicolegooden**
### Participating Group Members:

- Nicole Gooden
- Nathan Darrington

### Code Highlights:

- Set up `template` in `TheHome` and `TheHeader` components
- Set up `style` scoped to each component with basic styling/format
- Change `routes` by removing About and replacing Home with `TheHome` to match our new component

### Have Tests Been Added?

- [X] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?

I added some comments for future design / implementation.  
Links to `my parks` and `find park` don't actually render any components right now.  
We figured we would need to set up another component for the saved parks (as part of our MVP) that can be viewed via `my parks`.  The search component would render when `find park` is clicked.  We decided against the hamburger menu in the nav bar because we don't believe it's necessary for the header at this point, given there are only 2-3 links.  We can always add it back in if we get past our MVP before the deadline.

We handled mobile view only.

Please let me know if you imagine something different and we can work on adjusting our plan!

### Message/Questions for reviewer:

Do we want to set a special font for the Bone Yard title in the header?

### Issues:

- Addresses #6, #24
- Closes #28, #29

### Screenshots (if appropriate):

Screen Shot 2021-01-04 at 4 52 55 PM

### Tracking Consistency:

- [X] added appropriate labels
- [X] My code follows the code style of this project and has removed all unnecessary annotations
 (annotations in the code are placed there so that the comment is viewed with relevant code)
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas - N/A
- [ ] All new and existing tests passed - N/A

- [X] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/router/index.js
Comment (npdarrington) [inline src/router/index.js:11]: With the current PR that is also available from SP, we may run into a merge conflict with this file. We may have to decide which branch to merge first, pull the new main into the other branch and merge it together. It may not do that also so we will see how that goes!

File: src/App.vue
Comment (stacyp2006) [inline src/App.vue:5]: I'm excited to learn more about how Vue uses router. It seems simpler than React.

File: src/components/TheHeader.vue
Comment (npdarrington) [inline src/components/TheHeader.vue:12]: Looking at this again, with a mobile first approach this makes it look good! However, setting a `<br />` for each `router-link` for desktop may have some issues. We may want to assign a specific class for the `nav` and control it with flex and media queries. This way, we have more control over when the links have to go vertical and when they can go back to horizontal.

File: src/views/TheHome.vue
Comment (stacyp2006) [inline src/views/TheHome.vue:6]: I appreciate these comments. Helpful for whoever works on this code next.

**Review by stacyp2006 — APPROVED** [review]
Looks good!

**Review by npdarrington — COMMENTED** [review]
Thank you for cleaning up those extra files that were part of the `vue create` boiler plate!

**Review by npdarrington — APPROVED** [review]

File: src/App.vue
Comment (nicolegooden) [reply src/App.vue:5]: I agree! It looks very similar to how Redux was set up with router in our last project.

File: src/components/TheHeader.vue
Comment (nicolegooden) [reply src/components/TheHeader.vue:12]: 100% agree! I figure we'll take our those `<br />` tags when we get to styling more intentionally.  Definitely not a long-term solution by any means.

Comment (nicolegooden) [reply src/components/TheHeader.vue:12]: We'd probably just set a `display: block` for the `nav` so that each element within takes up its own space horizontally.

**Comment by nicolegooden** [comment]
@bretm9 No rush, feel free to rebase and merge this when you get a chance to review (unless we need to make some changes!)

File: src/router/index.js
Comment (bretm9) [reply src/router/index.js:11]: Ya, we can merge this and then rebase the other branch from main before merging. Should be an easy enough conflict to resolve!

**Review by bretm9 — APPROVED** [review]
Thanks for getting both TheHome and TheHeader done! These look great. Merging now.

**Pull request merged**
